README.md

```diff
@@ -159,6 +159,8 @@ extension to Python, you can then also set the shared linker environment variabl
 When invoked with the `--verbose` flag, verificarlo provides detailed output of
 the instrumentation process.
 
+If you want to use functions's instrumentation with a backend, you need to activate instrumentation at the compilation with the flag `--inst-func`. Be carefull the body of functions which are not compilated with verificarlo can't be modified in any way.
```

Comment:
This should go into a separate section specific for "Function instrumentation".

compilated -> compiled
carefull -> careful
(if you can please run a spell checker on your changes).

Reply:
I think I will use one, typos are corrected

Reply:
I've made a specific section for the instrumentation, she is short